components/user_settings/display/user_settings_display.tsx

```diff
@@ -35,40 +36,92 @@ function getDisplayStateFromProps(props) {
     };
 }
 
-export default class UserSettingsDisplay extends React.PureComponent {
-    static propTypes = {
-        user: PropTypes.object,
-        updateSection: PropTypes.func,
-        activeSection: PropTypes.string,
-        closeModal: PropTypes.func.isRequired,
-        collapseModal: PropTypes.func.isRequired,
-        setRequireConfirm: PropTypes.func.isRequired,
-        setEnforceFocus: PropTypes.func.isRequired,
-        timezones: PropTypes.array.isRequired,
-        userTimezone: PropTypes.object.isRequired,
-        allowCustomThemes: PropTypes.bool,
-        enableLinkPreviews: PropTypes.bool,
-        defaultClientLocale: PropTypes.string,
-        enableThemeSelection: PropTypes.bool,
-        configTeammateNameDisplay: PropTypes.string,
-        currentUserTimezone: PropTypes.string,
-        enableTimezone: PropTypes.bool,
-        shouldAutoUpdateTimezone: PropTypes.bool,
-        militaryTime: PropTypes.string,
-        teammateNameDisplay: PropTypes.string,
-        channelDisplayMode: PropTypes.string,
-        messageDisplay: PropTypes.string,
-        collapseDisplay: PropTypes.string,
-        linkPreviewDisplay: PropTypes.string,
-        lockTeammateNameDisplay: PropTypes.bool,
-        actions: PropTypes.shape({
-            getSupportedTimezones: PropTypes.func.isRequired,
-            autoUpdateTimezone: PropTypes.func.isRequired,
-            savePreferences: PropTypes.func.isRequired,
-        }).isRequired,
-    }
+type Option = {
+    value: string;
+    radionButtonText: {
+        id: string;
+        message: string;
+        moreId?: string;
+        moreMessage?: string;
+    };
+}
 
-    constructor(props) {
+type SectionProps ={
+    section: string ;
```

Comment:
Nitpick: extra space between the type and the semi-colon

Reply:
resolved it.